### PR TITLE
Feature/fftf 165 project items

### PR DESCRIPTION
**Pull request by ogbk**

### CODE REVIEW

File: app/routes/project-items.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
+const Joi = require('joi')
+const { setLabelData } = require('../helpers/helper-functions')
+
+function createModel (errorMessage, errorSummary, backLink, projectInfrastucture, projectEquipment, projectTechnology) {
```
Comment (diya2214) [inline app/routes/project-items.js:4]: remove error summary as not required on this page

File: app/routes/project-items.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  return {
+    backLink,
+    ...(errorSummary ? { errorText: errorSummary } : {}),
```
Comment (diya2214) [inline app/routes/project-items.js:7]: remove summary from here too

File: app/routes/project-items.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      const landOwnership = request.yar.get('landOwnership') || null
+      const backUrl = landOwnership === 'No' ? '/answers' : '/tenancy'
+
+      const projectInfrastucture = request.yar.get('projectInfrastucture') || null
+      const projectEquipment = request.yar.get('projectEquipment') || null
+      const projectTechnology = request.yar.get('projectTechnology') || null
+
+      return h.view(
+        'project-items',
+        createModel(null, null, backUrl, projectInfrastucture, projectEquipment, projectTechnology)
```
Comment (diya2214) [inline app/routes/project-items.js:93]: we do not need errorSummary argument here it should be **createModel(null, null, backUrl, projectInfrastucture, projectEquipment, projectTechnology)**

File: app/routes/project-items.js (hunk 2 above)
Comment (ogbk) [reply app/routes/project-items.js:7]: oh, I'm forgetting thing today :(

File: app/routes/project-items.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      validate: {
+        payload: Joi.object({
+          projectInfrastucture: Joi.any(),
+          projectEquipment: Joi.any(),
+          projectTechnology: Joi.any()
+        }),
+        failAction: (request, h) => {
+          const landOwnership = request.yar.get('landOwnership') || null
+          const backUrl = landOwnership === 'No' ? '/answers' : '/tenancy'
+          return h.view('project-items', createModel('Please select an option', null, backUrl, null)).takeover()
```
Comment (diya2214) [inline app/routes/project-items.js:110]: here too error summary argument not required

File: app/routes/project-items.js [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+          projectEquipment,
+          projectTechnology
+        } = request.payload
+
+        if (!projectInfrastucture && !projectEquipment && !projectTechnology) {
+          return h.view(
+            'project-items',
+            createModel(
+              'Select all the items your project needs',
+              null,
```
Comment (diya2214) [inline app/routes/project-items.js:128]: remove error summary argument from here too

**Review by diya2214 — APPROVED** [review]

**Pull request merged**